### ISSUE TITLE
Is YOLOv8 effective in recognizing horizontally flipped images?

### ISSUE THREAD

**Issue by Squidward1012** [ultralytics/ultralytics#2414]
### Search before asking

- [x] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

The dataset has been captured in multiple scenarios and after several training sessions, the model can perfectly recognize "turn around" and "stop" commands. However, it always confuses "turn left" and "turn right", recognizing both as the same command. How can this be solved? Can adjusting parameters help? Can someone please help to solve this issue?
![file_03](https://user-images.githubusercontent.com/95746383/236272665-da60c15b-0f74-434c-8f53-e17a23a807df.jpg)

**Comment by VdLMV** [comment]
My suggestion is to disable both vertical and horzontal flip augmentations and allow only small rotation angles during training.

**Comment by Squidward1012** [comment]
> My suggestion is to disable both vertical and horzontal flip augmentations and allow only small rotation angles during training.
I have already tired like this , but it didn't work T_T. Thanks for help.
![image](https://user-images.githubusercontent.com/95746383/236281360-79fbde56-bb2c-4c1d-beb0-44ae935cd70b.png)
![image](https://user-images.githubusercontent.com/95746383/236282420-67f29ec7-7cbb-48c4-adcc-bc1f3710bf1b.png)
![image](https://user-images.githubusercontent.com/95746383/236284906-5522e5cb-e711-4878-adf7-16bb6b38e3b0.png)
![image](https://user-images.githubusercontent.com/95746383/236284998-33e9bd29-a230-428b-b378-664867a8ee13.png)
dataset is like this
![image](https://user-images.githubusercontent.com/95746383/236281771-744e0c45-7894-4400-b6be-1d1ec9d21866.png)

**Comment by glenn-jocher** [comment]
@VdLMV, sorry to hear that disabling flip augmentations and enabling rotation angles didn't work for you. In this case, you may want to try to adjust the learning rate and batch size parameters and experiment with different values to see if it has any effect on improving the accuracy for distinguishing "turn left" and "turn right". Additionally, you can also try adding more training data specific to turning left and right, or experiment with different training methods such as transfer learning. Finally, it can be helpful to inspect the predicted output of the model on various examples of the confusing classes to see if there are any patterns that can help guide further adjustments to the model or its training process.

**Comment by VdLMV** [comment]
@glenn-jocher this is about @Squidward1012 's project,  I  was just giving my thoughts about his issue.

**Comment by glenn-jocher** [comment]
@VdLMV, thank you for sharing your thoughts on @Squidward1012's project. We always appreciate community involvement and collaboration in addressing YOLOv8 issues. @Squidward1012, if disabling flip augmentations and enabling rotation angles did not improve the accuracy of your model in distinguishing "turn left" and "turn right," I would suggest adjusting additional hyperparameters such as learning rate and batch size, and experimenting with different values in order to find the optimal combination. Increasing the amount of training data specific to the confusing classes and/or experimenting with different training methods such as transfer learning may also be beneficial. Finally, analyzing the predicted output of the model on different examples of the confusing classes may help guide further adjustments to the model or its training process.
